### PR TITLE
WIP Documentation changes

### PR DESCRIPTION
**Pull request by jamisonassuncao**
Changed incorrect image (there was an incorrect subscript value for the 2D temperature grid example); 
Changed param.txt file in the documentation (parameter file section);
Changed van Keken's Python script to create proper param.txt file (just to fit new param.txtx changes, nothing big).

### CODE REVIEW

File: docs/files/src/param.txt [hunk 1]
```diff
@@ -1,82 +1,111 @@
...
-# temperature_from_ascii  = True       # Use 'input_temperature_0.txt' ASCII file
-velocity_from_ascii       = False      # Use 'input_velocity_0.txt' ASCII file
-
-geo=on#kdjfkdjf
-geoq_fac                  = 100.0
-surf_veloc                = 0.0e-2.
-
-temp_diff                 = 0
-
+multigrid                           = 1             # ok -> soon to be on the command line only
```
Comment (victorsacek) [inline docs/files/src/param.txt:11]:
```suggestion

```

File: docs/files/src/param.txt [hunk 2]
```diff
@@ -12,15 +12,13 @@ RK4                                 = Euler         # particles advection method
...
-
-basal_heat                          = -1.0          # default is -1.0
+basal_heat                          = -1.0          # set basal heat flux value, default is -1.0 (i.e. not used) [W/m^2]
 
 # Particles options
-particles_per_element               = 1100          # default is 81
+particles_per_element               = 1100          # n. of Lagrangian particles in each element, default is 81
 particles_per_element_x             = 0             # n. of Lagrangian particles in the horizontal direction, default is 0 for automatic calculation
 particles_per_element_z             = 0             # n. of Lagrangian particles in the vertical direction, default is 0 for automatic calculation
 particles_perturb_factor            = 0.0           # default is 0.5 [values are between 0 and 1]
```
Comment (jamisonassuncao) [inline docs/files/src/param.txt:24]: @victorsacek Can you add a description to this parameter? I think we missed it during the discussion

File: docs/files/parameter-file.rst [hunk 3]
```diff
@@ -3,59 +3,416 @@
...
+    * particles_per_element_z             
+        default: 0 (automatic calculation) :raw-html:`<br />`
+        type: integer :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: number of Lagrangian particles in the vertical direction
+    * particles_perturb_factor            
+        default: 0.5  :raw-html:`<br />`
+        type: real number between 0 and 1 :raw-html:`<br />`
+        unit: :raw-html:`<br />`
+        definition: default is 0.5
```
Comment (jamisonassuncao) [inline docs/files/parameter-file.rst:113]: @victorsacek Can you add a description to this parameter? I think we missed it during the discussion

**Review by victorsacek — CHANGES_REQUESTED** [review]

File: docs/files/parameter-file.rst (hunk 3 above)
Comment (jamisonassuncao) [inline docs/files/parameter-file.rst:113]: ``
```suggestion
        definition: indicates the amount of perturbation of the initial location of the particles relative to a regular grid distribution. 
```

**Pull request merged**